src/languages/es.ts

```diff
@@ -855,6 +855,7 @@ const translations = {
         trackExpense: 'Seguimiento de gastos',
         chooseRecipient: 'Elige destinatario',
         confirmDetails: 'Confirma los detalles',
+        createExpenseWithAmount: ({amount}: {amount: string}) => `Crea un gasto de ${amount}`,
```

Comment:
Have we got a confirmation for the translation on this one? I tried using JaimeGPT in [the checklist](https://github.com/callstack-internal/Expensify-App/blob/103bf6cba97b85a4c30a78c07dc38c137cb9f4e0/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L88), and the result was `Crear un gasto de ${amount}.` instead of `Crea un gasto de ${amount}`

Reply:
I used the standard Chat GPT for it, but most likely you are right, I am going to change it!

Reply:
And don’t forget to post it in #expensify-open-source and `Link to Slack message` as per the [checklist item](https://github.com/callstack-internal/Expensify-App/blob/103bf6cba97b85a4c30a78c07dc38c137cb9f4e0/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L88)

Reply:
Done! Thanks!